docs/reference/argocd.md

```diff
@@ -964,16 +964,50 @@ There are two ways to customize resource behavior- the first way, only available
 
 ### Resource Customizations (with subkeys) Example
 
-Keys for `resourceHealthChecks`, `resourceIgnoreDifferences`, and `resourceActions` are in the form (respectively): `resource.customizations.health.<group_kind>`, `resource.customizations.ignoreDifferences.<group_kind>`, and `resource.customizations.actions.<group_kind>`. The following example defines a custom health check, custom action, and an ignoreDifferences config in the `argocd-cm` ConfigMap. 
+Keys for `resourceHealthChecks`, `resourceIgnoreDifferences`, and `resourceActions` are in the form (respectively): `resource.customizations.health.<group_kind>`, `resource.customizations.ignoreDifferences.<group_kind>`, and `resource.customizations.actions.<group_kind>`. The following example defines a custom health check, custom action, and an ignoreDifferences config in the `argocd-cm` ConfigMap. Additionally, `.spec.resourceIgnoreDifferences.all` allows you to apply these specified settings to all resources managed by this Argo CD instance.
 
 ``` yaml
 apiVersion: argoproj.io/v1alpha1
 kind: ArgoCD
 metadata:
-  name: example-argocd
-  labels:
-    example: resource-customizations
+  name: argocd
 spec:
+  resourceIgnoreDifferences:
+    all:
+      jqPathExpressions:
```

Comment:
I wasn't aware that these fields have been changed to slices. That is good. Thanks. Please realistic examples instead of xyz and abc.